lib/runner/variant-reassignment.js

```diff
@@ -550,7 +550,11 @@ export default class VariantReassignment {
       const stagedDraftSlugs = _.merge({}, productDraftSlug, slug)
       const stagedDraftSlugsLength = Object.keys(stagedDraftSlugs).length
 
-      if (slugLength + productDraftSlugLength !== stagedDraftSlugsLength)
+      const hasSameSlugLang
```

Comment:
Is this first condition `slugLength + productDraftSlugLength !== stagedDraftSlugsLength` necessary? Maybe it will work just with comparing lang values as you added in this PR:
```js
const hasAnySameSlugValue = Object.keys(slug)
  .some(lang => productDraftSlug[lang] === slug[lang])
```

Reply:
I tried it and it does not work.. so you can ignore this comment..